packages/boilerplate/composables/src/getters/cartHelpers.ts

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-unused-vars, @typescript-eslint/no-unused-vars */
+import { Cart } from '@vue-storefront/boilerplate-api/src/types';
+
+// TODO: Change UICartProduct to something agnostic.
+export const getProducts = (cart: Cart, includeAttributes: string[] = []): any[] => {
```

Comment:
includeAttributes should be in object. Overall
`getSomething(source, options)`
includeAttributes is in this case one of the properties here.
Also should be optional

Reply:
Outside the scope of the task